include/capo/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+target_sources(${PROJECT_NAME} PRIVATE
```

Comment:
Why do the headers need to be added as sources?

Reply:
It may not matter for editors like VSCode with clangd, but eg VS solutions won't show any files not explicitly part of the project. Also, they _are_ sources, even if not directly compiled.